physics/mp_nssl.meta

```diff
@@ -165,6 +165,30 @@
   type = real
   kind = kind_phys
   intent = in
+[nssl_alphar]
+  standard_name = nssl_alpha_rain
+  long_name = rain PSD shape parameter in NSSL micro
+  units = none
+  dimensions = ()
+  type = real
+  kind = kind_phys
+  intent = in
+[nssl_ehw0_in]
+  standard_name = nssl_hw_collec_eff
+  long_name = graupel droplet collection efficiency in NSSL micro
+  units = none
+  dimensions = ()
+  type = real
+  kind = kind_phys
+  intent = in
+[nssl_ehlw0_in]
+  standard_name = nssl_hlw_collec_eff
```

Comment:
Please change the standard name to `nssl_hail_collection_efficiency` here and in https://github.com/NOAA-EMC/fv3atm/pull/632